srcpkgs/visual-studio-code-bin/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'visual-studio-code-bin'
+pkgname=visual-studio-code-bin
+version=1.24.1
+revision=1
+only_for_archs="x86_64"
+wrksrc="${pkgname}-${version}"
+homepage="https://code.visualstudio.com"
+depends="alsa-lib GConf gtk+ libnotify libXtst nss python"
+short_desc="Editor for building and debugging modern web and cloud applications"
+maintainer="Leandro Vital <leavitals@gmail.com>"
+license="microsoft"
```

Comment:
According to github, the license is MIT.

Reply:
hmmm... why is their binary version different than their source version....

Reply:
This is the official distributed version, proprietary license

Reply:
Their own build is different from open-source version, they call open-source variant "vscode oss" or something and it might or might not lack some features. IIRC oss-version can't install extensions from the "shop" w/o some prior patching, maybe something else is different too (idk).